pkg/services/sqlstore/sqlstore.go

```diff
@@ -443,7 +443,7 @@ func (ss *SQLStore) ensureTransactionIsolationCompatibility(engine *xorm.Engine,
 	if errors.As(err, &mysqlError) {
 		// if there was an error due to transaction isolation
 		if strings.Contains(mysqlError.Message, "Unknown system variable 'transaction_isolation'") {
-			ss.log.Debug("transaction_isolation system var is unknown, overriding in connection string with tx_isolation instead")
+			ss.log.Debug("Transaction_isolation system var is unknown, overriding in connection string with tx_isolation instead")
```

Comment:
I think that `transaction_isolation` is an option so we need to keep it as is: maybe consider re-phrasing it